hail/src/test/scala/is/hail/DistributedSuite.scala

```diff
@@ -0,0 +1,36 @@
+package is.hail
+
+import is.hail.backend.distributed.DistributedBackend
+import is.hail.scheduler.Executor
+import is.hail.utils.TempDir
+import org.apache.hadoop
+import org.scalatest.testng.TestNGSuite
+import org.testng.annotations.BeforeClass
+
+object DistributedSuite {
+  def getSchedulerHost: String = {
+    val schedulerHost = System.getenv("HAIL_TEST_SCHEDULER_HOST")
+    if (schedulerHost == null) "localhost" else schedulerHost
+  }
+
+  lazy val hc: HailContext = {
+    val hc = HailContext.createDistributed(getSchedulerHost, logFile = "/tmp/hail.log")
+    if (System.getenv("HAIL_ENABLE_CPP_CODEGEN") != null)
+      hc.flags.set("cpp", "1")
+    hc.checkRVDKeys = true
+    hc
+  }
+}
+
+class DistributedSuite extends TestNGSuite {
```

Comment:
We probably eventually want to be able to run all the existing (distributed) tests against the Scheduler.  This forces us to statically pick which backend to use.  What about SparkSuite => HailSuite (or SuiteWithHailContext, or even ... DistributedSuite?)  And have a flag that selects the backend to construct.  Maybe use the scheduler backend if HAIL_TEST_SCHEDULER_HOST is defined?